src/main/scala/sigmastate/serialization/ErgoTreeSerializer.scala

```diff
@@ -127,15 +128,18 @@ class ErgoTreeSerializer {
     deserializeErgoTree(r)
   }
 
-  def deserializeErgoTree(r: SigmaByteReader): ErgoTree = {
+  def deserializeErgoTree(r: SigmaByteReader,
+                          maxTreeSizeBytes: Int = SigmaSerializer.MaxPropositionSize): ErgoTree = {
```

Comment:
Is this really a function parameter? Let's either remove it from parameters list or remove the default value

Reply:
@catena2w Probably not. Easing of testing does not count. Apart from that, decoupling from `SigmaSerializer` constant does not worth it alone. Removing default value will just force this constant leaking out. Let me remove the parameter.

Reply:
removed default value.